src/nvim/CMakeLists.txt

```diff
@@ -399,6 +399,10 @@ if(JEMALLOC_FOUND)
   list(APPEND NVIM_EXEC_LINK_LIBRARIES ${JEMALLOC_LIBRARIES})
 endif()
 
```

Comment:
Please also add a `USE_LTO` option (which defaults to ON), so that LTO can easily be avoided.

Reply:
Done